iosxr_iso2vbox.py

```diff
@@ -565,15 +578,7 @@ def main(argv):
     configure_xr(args.verbose)
 
     # Good place to stop and take a look if --debug was entered
-    if args.debug:
-        print("Stopping before creating the VM")
-        print("Use: 'socat TCP:localhost:65000 -,raw,echo=0,escape=0x1d' to access the VM")
-        raw_input("Press Enter to continue.")
-        # To debug post box creation, add the following lines to Vagrantfile
-        # config.vm.provider "virtualbox" do |v|
-        #   v.customize ["modifyvm", :id, "--uart1", "0x3F8", 4, "--uartmode1", 'tcpserver', 65005]
-        #   v.customize ["modifyvm", :id, "--uart2", "0x2F8", 3, "--uartmode2", 'tcpserver', 65006]
-        # end
+    pause_to_debug()
```

Comment:
I'd rather not turn debug into a global - perhaps instead:

```python
if args.debug:
    pause_to_debug() # no longer checks global `debug` since you wouldn't have called it if not desired
```